lib/index.js

```diff
@@ -58,6 +61,10 @@
             }.bind(this));
         }
 
+        var timeout = parseInt(this._options.timeout, 10) || 10000;
+        if (timeout < 0) timeout = 10000;
```

Comment:
Do you think we'll ever want to change this from 10s? Perhaps it would be good to use a constant for this value?